user/TASK/test_task/test_task.c

```diff
@@ -13,58 +12,88 @@
 #include "delay.h"
 #include "INS_task.h"
```

Comment:
This is the correct name of the header file, but Keil thinks it is INS_**T**ask.h. I think it would be better to get Keil to recognize the lowercase t file instead of just changing the include to a capital T, since we use lowercase for all other tasks.